src/main/java/bowling/model/gameresult/GameResult.java

```diff
@@ -0,0 +1,25 @@
+package bowling.model.gameresult;
+
+public class GameResult {
+
+    private final static int FINAL_FRAME_NO = 10;
```

Comment:
관례적으로 자바의 키워드의 순서를 따를 필요가 있습니다 ㅎ
```suggestion
    private static  final int FINAL_FRAME_NO = 10;
```